android/fastlane/Fastfile

```diff
@@ -60,15 +76,34 @@ platform :android do
 
   desc "Deploy to HockeyApp"
   lane :deploy_hockey do |options|
-    notes = sh("git log --format=%B -n 1 #{options[:commit_sha]}")
     hockey(
       api_token: options[:api_token],
       public_identifier: options[:app_id],
       apk: options[:apk_path],
-      notes: notes,
+      notes: options[:notes],
       status: options[:status],
       notify: options[:notify],
       commit_sha: options[:commit_sha]
     )
   end
+
+  desc "Upload to Beta by Fabric"
+  lane :deploy_fabric do |options|
+    crashlytics(
+      api_token: options[:api_token],
+      build_secret: options[:build_secret],
+      apk_path: options[:apk_path],
+      notes: options[:notes],
+      groups: options[:groups],
+      notifications: options[:notify]
+    )
+  end
+
+  def build_notes(commit_sha)
+    date = Time.now.strftime('%F')
+    time = Time.now.strftime('%T')
+    commit_msg = sh("git log --format=%B -n 1 #{commit_sha}").strip
+    notes = "#{git_branch}/#{commit_msg} on #{date} at #{time}"
```

Comment:
Made some changes to build notes format, open to suggestions to help make it simple to identify a build for testing.